### DISCUSSION TITLE
Dropdown attr aria-expanded missing because undefined value

### DISCUSSION THREAD

**Discussion by ramonrangelc** [primefaces/community#1209] [PrimeNG] [answered]
Hello we are using primeng v17.7.0 There is an issue with dropdowns accessibility. From dropdown.ts code overlayVisible is a Nullable<boolean> and [attr.aria-expanded]="overlayVisible" so by default aria-expanded is undefined, causing a critical accessibility issue. A simple undefine validation on dropdown template code should fix the issue: [attr.aria-expanded]="overlayVisible ?? false" 
![image](https://github.com/primefaces/community/assets/132338582/7eda1eab-f203-4dd1-94bf-96360f55895a)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for the feedback we'll address this issue in upcoming releases. Issue link: https://github.com/primefaces/primeng/issues/14832
